hooks/accordion.php

```diff
@@ -5,8 +5,15 @@
  * behind our fixed header. See: https://wordpress.org/plugins/accordion-shortcodes/other_notes/
  */
 add_filter('shortcode_atts_accordion', function($atts) {
-    if (isset($atts['scroll']) && $atts['scroll']) {
+    if (isset($atts['scroll']) || $atts['scroll']) {
```

Comment:
derp. correction in [latest commit](https://github.com/netrivet/turbo-support/pull/5/commits/41b8664f1583135b9d81730395bb2dbcb37a4e44).